packages/pangea-node-sdk/src/types.ts

```diff
@@ -2175,42 +2175,112 @@ export namespace Share {
   export type Tags = string[];
 
   export interface ItemData {
+    /**
+     * The number of billable bytes (includes Metadata, Tags, etc.) for the
+     * object.
+     */
+    billable_size?: number;
+
+    /** The date and time the object was created. */
+    created_at: string;
+
+    /** The ID of a stored object. */
     id: string;
-    type: string;
+
+    /** The MD5 hash of the file contents. */
+    md5?: string;
+
+    /**
+     * A set of string-based key/value pairs used to provide additional data
+     * about an object.
+     */
+    metadata?: Metadata;
     name: string;
-    created_at: string;
-    updated_at: string;
     parent_id: string;
-    size?: number;
-    billable_size?: number;
-    location?: string;
-    tags?: Tags;
-    metadata?: Metadata;
-    md5?: string;
+
+    /** The SHA256 hash of the file contents. */
     sha256?: string;
+
+    /** The SHA512 hash of the file contents. */
     sha512?: string;
+
+    /** The size of the object in bytes. */
+    size?: number;
+
+    /** A list of user-defined tags. */
+    tags?: Tags;
+
+    /** The type of the item (file or dir). */
+    type: string;
+
+    /** The date and time the object was last updated. */
+    updated_at: string;
   }
 
   export interface DeleteRequest {
+    /**
+     * The ID of the object to delete.
+     */
     id?: string;
+
+    /**
+     * If true, delete a folder even if it's not empty. Deletes the contents of
+     * folder as well.
+     */
     force?: boolean;
+
+    /**
+     * The path of the object to delete.
+     */
     path?: string;
   }
 
   export interface Authenticator {
+    /**
+     * An authentication mechanism.
+     */
     auth_type: AuthenticatorType;
+
+    /**
+     * An email address.
+     */
     auth_context: string;
   }
 
   export interface DeleteResult {
+    /**
+     * Number of objects deleted.
+     */
     count: number;
   }
 
   export interface FolderCreateRequest {
+    /**
+     * The name of an object.
+     */
     name?: string;
+
+    /**
+     * A set of string-based key/value pairs used to provide additional data
+     * about an object.
+     */
     metadata?: Metadata;
+
+    /**
+     * The ID of a stored object.
+     */
     parent_id?: string;
+
+    /**
+     * An case-sensitive path to an object. Contains a sequence of path segments
```

Comment:
```suggestion
     * A case-sensitive path to an object. Contains a sequence of path segments
```

Reply:
I blame [the docs](https://pangea.cloud/docs/api/share#/v1beta/folder/create)! 😄 
Fixed here now.